### PR TITLE
Add documentation on how to use the docker image

### PR DESCRIPTION
**Pull request by felipecrs**
References #1134

### CODE REVIEW

**Comment by felipecrs** [comment]
Do we need really to mount the `helm` configuration (`~/.helm` for v2 and `~/.config/helm` for v3)? Don't the `helmfile.yaml` itself already contain all the needed configuration?

File: Dockerfile [hunk 1]
```diff
@@ -46,4 +46,6 @@ RUN helm plugin install https://github.com/databus23/helm-diff && \
 
 COPY --from=builder /workspace/helmfile/dist/helmfile_linux_amd64 /usr/local/bin/helmfile
 
-CMD ["/usr/local/bin/helmfile"]
+WORKDIR /wd
```
Comment (mumoshu) [inline Dockerfile:49]: This makes sense, but I'd like to keep this `WORKDIR /`, as I've seen someone uses the image like:

```
docker -v ${PWD}/helmfile.yaml:/helmfile.yaml ...
```

And this change can be a breaking change for those.

File: README.md [hunk 2]
```diff
@@ -365,12 +365,40 @@ If you wish to treat your enviroment variables as strings always, even if they a
...
 - Windows (using [scoop](https://scoop.sh/)): `scoop install helmfile`
 - macOS (using [homebrew](https://brew.sh/)): `brew install helmfile`
 
+### Running as a container
+
+The [Helmfile Docker images are available in Quay](https://quay.io/roboll/helmfile). There is no `latest` tag, since the `0.x` versions can contain breaking changes, so make sure you pick the right tag. Example using `helmfile 0.135.0`:
+
+```sh-session
+# helm 2
+$ docker run --rm --net=host -v "${HOME}/.kube:/root/.kube" -v "${HOME}/.helm:/root/.helm" -v "${PWD}:/wd" quay.io/roboll/helmfile:v0.135.0 helmfile sync
```
Comment (mumoshu) [inline README.md:380]:
```suggestion
$ docker run --rm --net=host -v "${HOME}/.kube:/root/.kube" -v "${HOME}/.helm:/root/.helm" -v "${PWD}:/wd" --workdir /wd quay.io/roboll/helmfile:v0.135.0 helmfile sync
```

This does make sense, but as I've commented at https://github.com/roboll/helmfile/pull/1607/files#r540608083, I'd like to use `/` as the default workdir and explicitly set `--workdir /wd` for this "recommended" use-case.

I ABSOLUTELY agree we should have a great default, but I believe the image is already used widely in production and our users won't love breaking changes :)

File: Dockerfile (hunk 1 above)
Comment (felipecrs) [reply Dockerfile:49]: Indeed this is a breaking change for some. However, helmfile supports referencing multiple helmfiles, and mounting one by one can be tricky.

File: README.md (hunk 2 above)
Comment (felipecrs) [reply README.md:380]: Makes sense, to prevent a breaking change. Perhaps in the future. :)

**Review by mumoshu — APPROVED** [review]
LGTM. Thank you so much for your contribution @felipecrs!

**Pull request merged**